tests/test_graph.py

```diff
@@ -15,11 +14,14 @@ def test_filter_graph(self):
                                             |
                                             ------<Scale>--[O/720p]
         """
-        logo = inputs.Input(inputs.Stream(VIDEO), input_file='logo.png')
-        main = inputs.Input(input_file='main.mp4')
-        il = inputs.InputList(main, logo)
+        logo = Input(Stream(VIDEO), input_file='logo.png')
+        main = Input(input_file='main.mp4')
+        il = InputList(main, logo)
+        out0 = Output('out0.mp4', Codec(VIDEO, 'v'), Codec(AUDIO, 'a'))
+        out1 = Output('out1.mp4', Codec(VIDEO, 'v'), Codec(AUDIO, 'a'))
```

Comment:
Also there should be possibility to pass only filename - empty stubs will be added automatically.